python/queries.gql

```diff
@@ -91,3 +91,33 @@ subscription Updates($sessionId: ID!, $clientId: ID!) {
     }
   }
 }
+
+
+fragment CollectionReference on Collection {
+      id
+      key
+}
+
+fragment CollectionNotFound on CollectionNotFound {
+      id
+}
+
+
+mutation CollectionCreate(
+  $organizationID: ID!
+  $key: ID!
+   $parentID: ID
+) {
+  collectionCreate(
+    organizationID: $organizationID
+    key: $key
+    parentID: $parentID
+  ) {
+    __typename
+     ... on Collection {
```

Comment:
Fix the formatting here - find an autoformatter and use that :) try the plugin in vscode